src/format/mod.rs

```diff
@@ -485,6 +485,51 @@ impl Locales {
     }
 }
 
+/// Type for specifying the format of UTC offsets.
```

Comment:
This should be in the same commit where it starts being used. I think I'd prefer it just being called `OffsetFormat` -- the `Utc` prefix seems superfluous.